frontend/pages/hosts/details/cards/HostSummary/HostSummary.tsx

```diff
@@ -129,13 +132,20 @@ const HostSummary = ({
   );
 
   const renderSummary = () => {
+    const hostStatus = titleData.status;
     return (
       <div className="info-flex">
         <div className="info-flex__item info-flex__item--title">
           <span className="info-flex__header">Status</span>
-          <span className={`${statusClassName} info-flex__data`}>
-            {titleData.status}
-          </span>
+          <StatusCell
+            value={hostStatus}
+            // TODO: setting id to "status-tooltip" is fine here because there is only one status
```

Comment:
you could make the id parameter optional and remove the id key here?

Reply:
I _think_ an id is required for the tooltip to function.

Not sure I understand the `TODO` in the code. First it says it's okay to be `status-tooltip`, then says it'd be better if it was the host's ID? Seems like we should make a decision one way or the other and remove the TODO.